### PR TITLE
Impactdevs blog

### PR DESCRIPTION
**Pull request by JessicaML**

### CODE REVIEW

File: blog/_posts/2017-07-31-team-impactdevs-part-one.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Team ImpactDevs Amsterdam - Part 1
+layout: post
+created_at: Tue Jul 31 2017
```
Comment (alicetragedy) [inline blog/_posts/2017-07-31-team-impactdevs-part-one.md:4]: not a big deal but this should say `Mon` — I will change it myself later!

File: blog/_posts/2017-07-31-team-impactdevs-part-one.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+permalink: blog/2017-07-31-team-impactdevs
+current: blog
+author: Jess
+twitter: teamimpactdevs
+categories:
+- student-posts
+---
+
+![image](../img/blog/2017/2017-07-teamimpactdevs.jpg)
+<div class="image-credits">Team ImpactDevs</div>
```
Comment (alicetragedy) [inline blog/_posts/2017-07-31-team-impactdevs-part-one.md]: could you add here an image credit for who took the picture/who it belongs to/who should be credited if we use it again? (e.g. `(Image: Team ImpactDevs)` or `(Image: Name-of-your-coach)`)

**Comment by JessicaML** [comment]
thanks Laura

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Thanks for your post @JessicaML! it's almost ready to merge, I'd just need you to add a line with a specific image credit (see my comment). 🎉

**Comment by JessicaML** [comment]
all done!

**Review by alicetragedy — APPROVED** [review]
Hooray! congratulations on your first contribution to our blog, @JessicaML ✨ 💪 💻

**Pull request merged**